### DISCUSSION TITLE
Reopening Tabs at the Same Location that where they were when they were Closed

### DISCUSSION THREAD

**Discussion by protean-onion** [TriliumNext/Trilium#3993] [Ideas]
Hello,

Let's say we have nine tabs open. Now let's say we close the fourth tab accidentally while dragging it. So we'd want to reopen it. We press `Ctrl + Shift + T`. We'd expect it to open right there, like a browser, but it opens at the end. Now you have to drag it back. Though that's alright ([inconvenience can be good](https://www.ft.com/content/0d206e43-26c9-4704-aab6-0783a20b83ad)), it'd be great if it opened at the same location in the same order.

Thanks for your time!

**Comment by meichthys** [comment]
I haven't noticed that before, but yes, it seems like it would be worth filing this in the issue tracker.

**Comment by Nriver** [comment]
Check this out https://github.com/SiriusXT/trilium-back-to-history

You can find more interesing extensions here https://github.com/Nriver/awesome-trilium

**Comment by dymani** [comment]
This would be very useful! I managed to get this feature working for split panes and multi-split tab as well, you guys may take a look at #4001
